cluster/ephemeral-provider-common.sh

```diff
@@ -29,11 +29,18 @@ function _registry_volume() {
 }
 
 function _add_common_params() {
-    local params="--nodes ${KUBEVIRT_NUM_NODES} --memory 4096M --cpu 4 --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
+    local params="--nodes ${KUBEVIRT_NUM_NODES} --cpu 4 --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
     if [[ $TARGET =~ windows.* ]]; then
-        params="--memory 8192M --nfs-data $WINDOWS_NFS_DIR $params"
```

Comment:
...and then you probably doesn't have to mess with the size here at all, and just set it to 8GB in automation/test.sh for windows target

Reply:
From my point, setting the variable in [test.sh](https://github.com/kubevirt/kubevirt/blob/00321ac43633411eec02c36a1aa5f4870acc4a16/automation/test.sh#L99), where defaults for windows are already set, is more readable for future changes.